apps/framework-cli/src/cli/routines.rs

```diff
@@ -332,12 +332,9 @@ async fn manage_leadership_lock(
             IS_RUNNING_LEADERSHIP_TASKS.store(false, Ordering::SeqCst);
         });
 
-        let mut client = redis_client.lock().await;
```

Comment:
Do we still need the lock here?

Reply:
Good catch, this wasn't strictly needed, now that that broadcast_messge is thread safe.